DigitalLearningSolutions.Web/Controllers/ForgotPasswordController.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace DigitalLearningSolutions.Web.Controllers
+{
+    using Microsoft.AspNetCore.Mvc;
+    using DigitalLearningSolutions.Web.ViewModels.ForgotPassword;
+    using DigitalLearningSolutions.Web.ViewModels.LearningSolutions;
+
+    public class ForgotPasswordController : Controller
+    {
+
+        [Route("/ForgotPassword")]
+        public IActionResult ForgotPassword()
```

Comment:
The Index method on the controller should have a "/ForgotPassword" route by default, so I think it'll be fine to name this Index. Also, it'll make it really obvious that this is the main forgot password page (both in controller and view folders). :)